pkg/notify/notify.go

```diff
@@ -0,0 +1,27 @@
+package notify
+
+import (
+    "os"
+    "time"
+    "encoding/json"
+    "strconv"
+    "github.com/slack-go/slack"
+)
+
+func Notify(msg_text string) error {
+    attachment := slack.Attachment{
+		Text:          msg_text,
+		FooterIcon:    "https://platform.slack-edge.com/img/default_application_icon.png",
+		Ts:            json.Number(strconv.FormatInt(time.Now().Unix(), 10)),
+	}
+	msg := slack.WebhookMessage{
+        Username: "a9s-aha-cli",
```

Comment:
Make it a const at the beginning of the file so it's easier to spot the hard coded values.